agent/stats/queue.go

```diff
@@ -103,6 +103,13 @@ func (queue *Queue) add(rawStat *ContainerStats) {
 			stat.CPUUsagePerc = 100 * cpuUsageSinceLastStat / timeSinceLastStat
 		}
 
+		//calculate per second Network metrics
+		timeSinceLastStatInSeconds := uint64(rawStat.timestamp.Sub(lastStat.Timestamp).Seconds())
+		if stat.NetworkStats != nil && timeSinceLastStatInSeconds > 0 {
+			stat.NetworkStats.RxBytesPerSecond = uint64(((stat.NetworkStats.RxBytes - lastStat.NetworkStats.RxBytes) / timeSinceLastStatInSeconds))
+			stat.NetworkStats.TxBytesPerSecond = uint64(((stat.NetworkStats.TxBytes - lastStat.NetworkStats.TxBytes) / timeSinceLastStatInSeconds))
```

Comment:
Will be sent for only bridge network mode? If not, we should add a check to ensure that this is only for bridge network mode task.

Reply:
That check is being done here already https://github.com/aws/amazon-ecs-agent/blob/master/agent/stats/engine.go#L616 . That is why I am not doing it here